src/Gui/Application.cpp

```diff
@@ -2248,7 +2245,7 @@ void Application::runApplication()
 #endif
 
     StartupPostProcess postProcess(&mw, app, &mainApp);
-    postProcess.execute();
+    postProcess.execute(false);
```

Comment:
In terms of clean code this change is questionable because the meaning of "false" or "true" is not transparent (the so called boolean blindness). One has to first check the implementation to see what it means.

Then it's better to define an enum class with two labels FromPython and FromFreeCAD

Reply:
You are right. Although I have another refactor attempt in queue where is flag is not needed at all, it is worth to fix it.

Reply:
I dropped that commit and made it part of future cleanup. It had no useful effect for now anyway. Sorry for the noise.